starlette/requests.py

```diff
@@ -60,7 +59,7 @@ class ClientDisconnect(Exception):
     pass
 
 
-class HTTPConnection(Mapping):
+class HTTPConnection(typing.Mapping[str, typing.Any]):
```

Comment:
I remember trying this before and it not working. I think it broke on Python 3.6. But now that we've dropped 3.6 we can remove 1 import and get better typing as well 🥳